runtime/include/tasks/qthreads/tasks-qthreads.h

```diff
@@ -134,13 +134,6 @@ typedef struct {
           .requestedSubloc = _subloc, \
           .prvdata = { .serial_state = _serial } }
 
-typedef struct {
-    void                     *fn;
-    void                     *args;
-    chpl_bool                countRunning;
-    chpl_task_prvDataImpl_t  chpl_data;
-} chpl_qthread_wrapper_args_t;
-
```

Comment:
What's the reason for moving this to the .c? I'm guessing we simply don't need it in the .h anymore, and it was a cleanup to move it, but I wanted to check that I'm not missing something